app/jobs/user_stat_collection_job.rb

```diff
@@ -3,8 +3,6 @@ class UserStatCollectionJob < ApplicationJob
   queue_as :default
 
   def perform(*_args)
-    return unless UserStat.exists?
```

Comment:
Not blocking, but I think we wanted to move this down to after importer.import instead of completely removing it.